src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine{
+
+    @Override
+    public void doWork() {
+        System.out.println("Machine started its work.");

```

Comment:
According to the checklist, the message should indicate what type of `Machine` is working. Consider changing the message to specify that a `Bulldozer` is working.